.github/workflows/pull-request-checks.yml

```
@@ -275,12 +279,41 @@ jobs:
         with:
           command: audit
 
+  # Only run check if the project still compiles on MSRV, test are run on stable
+  check-msrv:
+    name: Run `cargo check` on MSRV
+    runs-on: Ubuntu-20.04
+    needs: changes
+    if: ${{ needs.changes.outputs.rust == 'true' }}
```

Make this job run also when workflow is modified.
```suggestion
    if: ${{ needs.changes.outputs.rust == 'true' || needs.changes.outputs.workflows == 'true' }}
```